aerogear/unifiedpush-admin-ui
# [WIP] Rhmap 21183

Migrate from bower to yarn because bower is deprecated. This removes the need for bower and bower.config by moving all the dependencies into package.json as Github dependencies. This forces us to use yarn because some of those dependencies cannot be handled by npm (missing package.json). Bower is aware of this and recommends to use yarn: https://bower.io/blog/2017/how-to-migrate-away-from-bower/

This is the first step. Further steps should remove the Github dependencies one by one and replace them with proper npm packages where available. Possibly also upgrade them to newer / alternative versions. 

Verification steps:

1. Check out this branch and build the admin ui with `mvn clean install`
1. On your clone of UPS change the admin-ui version to 2.0.2-SNAPSHOT here: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/pom.xml#L214
1. Build UPS with `mvn clean install`
1. Run the generated container: `docker run -p 8080:8080 -it aerogear/ups:plain`
1. Open `localhost:8080` in a Browser. The admin-ui should work as usual
1. Create a Variant, etc. everything should work as usual
